docs/static_site/src/pages/get_started/ubuntu_setup.md

```diff
@@ -465,51 +451,37 @@ Or you can go through a manual process described next.
 
 **Build the MXNet core shared library**
 
-**Step 1** Install build tools and git.
-```bash
-$ sudo apt-get update
-$ sudo apt-get install -y build-essential git
-```
-
-**Step 2** Install OpenBLAS.
-
-*MXNet* uses [BLAS](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms) and [LAPACK](https://en.wikipedia.org/wiki/LAPACK) libraries for accelerated numerical computations on CPU machine. There are several flavors of BLAS/LAPACK libraries - [OpenBLAS](https://www.openblas.net/), [ATLAS](http://math-atlas.sourceforge.net/) and [MKL](https://software.intel.com/en-us/intel-mkl). In this step we install OpenBLAS. You can choose to install ATLAS or MKL.
-```bash
-$ sudo apt-get install -y libopenblas-dev liblapack-dev
-```
+**Step 1** Install R, cran-devtools and roxygen2
 
-**Step 3** Install OpenCV.
+If you are on Ubuntu 19.04 or higher:
 
-*MXNet* uses [OpenCV](https://opencv.org/) for efficient image loading and augmentation operations.
 ```bash
-$ sudo apt-get install -y libopencv-dev
+$ sudo apt-get update
+$ sudo apt-get install -y r-base-core r-cran-devtools r-cran-roxygen2
```

Comment:
would you please hint me on what's the issue for `roxygen2` on ubuntu 19.04?

Reply:
It's not included in the ubuntu repo prior to ubuntu 19.04. So users on older systems need to install it manually.